src/main/java/uk/gov/justice/hmpps/prison/web/config/SwaggerConfig.java

```diff
@@ -67,7 +65,8 @@ public Docket docket() {
                 .directModelSubstitute(LocalDateTime.class, Date.class)
                 .directModelSubstitute(LocalDate.class, java.sql.Date.class)
                 .securityContexts(Lists.newArrayList(securityContext()))
-                .securitySchemes(Lists.newArrayList(apiKey));
+                .securitySchemes(Lists.newArrayList(apiKey))
+                .forCodeGeneration(true);
```

Comment:
required to stop classes using inheritance being named with invalid characters

e.g. before we would have `Page«PrisonerInformation»`

after we have `PageOfPrisonerInformation`